scripts/setup-kustomize.sh

```diff
@@ -73,6 +73,9 @@ then
   source $USER_CONFIG_FILE
 fi
 
+if [ ! -z "${VERTICA_DEPLOYMENT_METHOD_OVERRIDE}" ]; then
+    VERTICA_DEPLOYMENT_METHOD=$VERTICA_DEPLOYMENT_METHOD_OVERRIDE
+fi
```

Comment:
I'm not sure if I like this. I understand why you did this because my instructions for using the new deployment method didn't work. 

I think to fix the earlier work I did we should:

1. Update tests/kustomize-defaults.cfg with this change so we don't set the variable.
```
# The type of deployment to use when running vertica admin commands. Valid
# values are: admintools or vclusterops. Uncomment this or set this environment
# in your shell to control the deployment type.
# VERTICA_DEPLOYMENT_METHOD=
```

2. Update the above line with following:
```suggestion
if [ -z "${VERTICA_DEPLOYMENT_METHOD}" ]; then
    VERTICA_DEPLOYMENT_METHOD=admintools
fi
```
3. Export VERTICA_DEPLOYMENT_METHOD in the e2e-udx.yml file